cfgov/jinja2/v1/activity-log/_activity-list.html

```diff
@@ -23,12 +23,12 @@
                     {% if is_blog(post) %}
                         {{ category_slug.render('blog',
                                 page_url,
-                                'meta-header_left',
+                                'm-meta-header_left',
```

Comment:
Added! Also, misread your original comment and I tried adding a `m-meta-header` and that changes the layout too.